src/bindings/python/tests_compatibility/test_inference_engine/test_IECore.py

```diff
@@ -293,6 +293,11 @@ def detect_long_gil_holds():
 
 
 def test_nogil_safe(device):
+    libc_name, libc_version = platform.libc_ver()
+    version = tuple(int(x) for x in libc_version.split('.'))
+    if libc_name == 'glibc' and version < (2, 34):
+        pytest.skip("There is an issue in glibc for elder version.")
```

Comment:
```suggestion
        pytest.skip("There is an issue in glibc for older version.")
```
Just a nit, I don't really mind if it's changed.

Reply:
updated